script/bash/convertCSVjson.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
please don't do this going forward.

We have no use for scripts converting CSV to json.

Ask the team to provide JSON, we have no reason to have all this code produced.